### PR TITLE
*: update storage unit in system variables and config file

### PR DESCRIPTION
**Pull request by lilin90**
### What is changed, added or deleted? (Required)

Modify the storage units in [System Variables](https://docs.pingcap.com/tidb/stable/system-variables) & [TiKV Configuration File](https://docs.pingcap.com/tidb/stable/tikv-configuration-file) from KB/MB/GB/TB to KiB/MiB/GiB/TiB

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/16844
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by lilin90** [comment]
@easonn7 PTAL. Thanks!

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -1214,14 +1214,14 @@ Configuration items related to RocksDB
 + Enables the readahead feature during RocksDB compaction and specifies the size of readahead data. If you are using mechanical disks, it is recommended to set the value to 2MB at least.
```
Comment (qiancai) [inline tikv-configuration-file.md:1214]:
```suggestion
+ Enables the readahead feature during RocksDB compaction and specifies the size of readahead data. If you are using mechanical disks, it is recommended to set the value to 2MiB at least.
```

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -1775,29 +1775,29 @@ Configuration items related to `raftdb`
...
 + Minimum value: `0`
-+ Unit: B|KB|MB|GB
++ Unit: B|KiB|MiB|GiB
 
 ### `max-total-wal-size`
 
 + The maximum RocksDB WAL size in total
-+ Default value: `"4GB"`
-    + When `storage.engine="raft-kv"`, the default value is `"4GB"`.
++ Default value: `"4GiB"`
```
Comment (qiancai) [inline tikv-configuration-file.md:1783]:
```suggestion
+ Default value:
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**